api/templates/api.mustache

```diff
@@ -388,11 +388,11 @@ func (a *{{{classname}}}Service) {{nickname}}Execute(r {{#structPrefix}}{{&class
 			var v {{{dataType}}}
 			err = a.client.decode(&v, localVarBody, localVarHTTPResponse.Header.Get("Content-Type"))
 			if err != nil {
-				newErr.error = _fmt.Sprintf("%s%v", errorPrefix, err.Error())
+				newErr.error = _fmt.Sprintf("%s: %s", newErr.Error(), err.Error())
```

Comment:
This change is to continue to build on the error message from `newErr` above; which contains the relevant error prefix (if any), and the response status like `404 Not Found`.

Reply:
Also changed the `%v` to be a `%s` to be consistent with the `Sprintf`'s elsewhere, since we are dealing with strings.